tests/test_rocket.py

```diff
@@ -8,88 +8,27 @@
-        power_off_drag="data/calisto/powerOffDragCurve.csv",
-        power_on_drag="data/calisto/powerOnDragCurve.csv",
-        center_of_mass_without_motor=0,
-        coordinate_system_orientation="tail_to_nose",
-    )
-
-    test_rocket.add_motor(test_motor, position=-1.255 - 0.1182359460624346)
-
-    test_rocket.set_rail_buttons(0.2 - 0.1182359460624346, -0.5 - 0.1182359460624346)
-
-    nose_cone = test_rocket.add_nose(
-        length=0.55829,
-        kind="vonKarman",
-        position=1.278 - 0.1182359460624346,
-        name="NoseCone",
-    )
-    fin_set = test_rocket.add_trapezoidal_fins(
-        4,
-        span=0.100,
-        root_chord=0.120,
-        tip_chord=0.040,
-        position=-1.04956 - 0.1182359460624346,
-    )
-    tail = test_rocket.add_tail(
-        top_radius=0.0635,
-        bottom_radius=0.0435,
-        length=0.060,
-        position=-1.194656 - 0.1182359460624346,
-    )
-
-    def drogue_trigger(p, h, y):
-        # p = pressure
-        # y = [x, y, z, vx, vy, vz, e0, e1, e2, e3, w1, w2, w3]
-        # activate drogue when vz < 0 m/s.
-        return True if y[5] < 0 else False
-
-    def main_trigger(p, h, y):
-        # p = pressure
-        # y = [x, y, z, vx, vy, vz, e0, e1, e2, e3, w1, w2, w3]
-        # activate main when vz < 0 m/s and z < 800 m.
-        return True if y[5] < 0 and h < 800 else False
-
-    main = test_rocket.add_parachute(
-        "Main",
-        cd_s=10.0,
-        trigger=main_trigger,
-        sampling_rate=105,
-        lag=1.5,
-        noise=(0, 8.3, 0.5),
-    )
-
-    drogue = test_rocket.add_parachute(
-        "Drogue",
-        cd_s=1.0,
-        trigger=drogue_trigger,
-        sampling_rate=105,
-        lag=1.5,
-        noise=(0, 8.3, 0.5),
-    )
-
+def test_rocket(mock_show, calisto_robust):
+    test_rocket = calisto_robust
     static_margin = test_rocket.static_margin(0)
-
     # Check if all_info and static_method methods are working properly
     assert test_rocket.all_info() == None or not abs(static_margin - 2.05) < 0.01
-    # Check if NoseCone all_info() is working properly
-    assert nose_cone.all_info() == None
-    # Check if fin_set all_info() is working properly
-    assert fin_set.all_info() == None
-    # Check if tail all_info() is working properly
-    assert tail.all_info() == None
-    # Check if draw method is working properly
-    assert fin_set.draw() == None
 
 
 @patch("matplotlib.pyplot.show")
-def test_coordinate_system_orientation(mock_show):
+def test_aero_surfaces_infos(
+    mock_show, calisto_nose_cone, calisto_tail, calisto_trapezoidal_fins
+):
+    assert calisto_nose_cone.all_info() == None
+    assert calisto_trapezoidal_fins.all_info() == None
+    assert calisto_tail.all_info() == None
+    assert calisto_trapezoidal_fins.draw() == None
+
+
+@patch("matplotlib.pyplot.show")
+def test_coordinate_system_orientation(
+    mock_show, calisto_nose_cone, calisto_tail, calisto_trapezoidal_fins
+):
```

Comment:
Ok ok, I started to use the fixture as you suggested. But I thought it would be safer to use not change the csys attribute of the second rocket, instead I created other instance.